site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
+        SEO Keywords: Erasure coding, inline, forward error correction, reed-solomon code, silent data corruption, RAID6, BitRot protection, healing, highway hash
 
       - Distributed Minio Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/distributed/README.md
         Slug: distributed-minio-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Learn more about Deploying MinIO in Distributed Mode
+        SEO Description: The MinIO Distributed Code Quickstart Guide offers examples, code and insight on how to pool multiple dives into a single object storage server
+        SEO Keywords: distributed mode, pooling drives, Consistency Guarantees
 
       - How to secure access to Minio server with TLS:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/tls/README.md
         Slug: how-to-secure-access-to-minio-server-with-tls
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Learn how to secure access to MinIO server with TLS
+        SEO Description: This guide explains how to configure MinIO Server with TLS certificates on Linux and Windows platforms.
+        SEO Keywords: TLS certificates, private key and public certificates, certificate authority, self-signed keys, OpenSSL keys, ECDSA private keys, RSA private key, GnuTLS certificates
 
       - Minio Bucket Notification Guide:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/bucket/notifications/README.md
         Slug: minio-bucket-notification-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Learn how to use MinIO Bucket Notifications
+        SEO Description: The guide explains what events occuring on objects in buckets can be monitored using bucket event notifications
+        SEO Keywords: s3:ObjectCreated:Put, s3:ObjectCreated:CompleteMultipartUpload, s3:ObjectAccessed:Head, s3:ObjectCreated:Post, s3:ObjectRemoved:Delete, s3:ObjectCreated:Copy, s3:ObjectAccessed:Get, AMQP, RabbitMQ, RabbitMQ, Elasticsearch, Redis, NATS, postgres, mysql, Kafka, webhooks, nsq
 
       - Minio Server Limits Per Tenant:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/minio-limits.md
         Slug: minio-server-limits-per-tenant
-        SEO Title:
-        SEO Description:
+        SEO Title: MinIO | Learn how to configure your MinIO server
+        SEO Description: This guide includes code, examples and tips on how to get the most out of your MinIO server
         SEO Keywords:
 
       - Minio Server Configuration Guide:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/config/README.md
         Slug: minio-server-configuration-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Learn how to configure your minio server
```

Comment:
Is the lowercase `minio` intentional here?

Reply:
Taken from @jtsymonds text. @jtsymonds  can you please clarify. Maybe we should make this MinIO?

Reply:
Yes, MinIO